shared/gh/partials/agenda-view.html

```diff
@@ -0,0 +1,108 @@
+<%
+    prevDate = null;
+
+    /**
+     * Compare two dates to each other and return true if the year, month and day match each other
+     *
+     * @param  {Date}    _prevDate    The first Date to match against
+     * @param  {Date}    currDate     The second Date to match against
+     * @return {Boolean}              Return `true` if the dates match
+     * @private
+     */
+    var isADifferentDay = function(_prevDate, currDate) {
+        if (moment(_prevDate).format('YYYYMMDD') === moment(currDate).format('YYYYMMDD')) {
+            return true;
+        }
+        return false;
+    };
+
+    /**
+     * Get the current academic week number based on todays date
+     *
+     * @return {Number}    The current academic week number
+     */
+    var getCurrentAcademicWeekNumber = function() {
+        return data.utils.getAcademicWeekNumber(data.utils.convertISODatetoUnixDate((moment(new Date()).format('YYYY-MM-DD'))), true);
+    };
+
+    /**
+     * Whether or not the provided date is today
+     *
+     * @param  {Date}       date    The date to check
+     * @return {Boolean}            Returns `true` if the provided date is today
+     * @private
+     */
+    var isToday = function(date) {
+        if (moment(date).format('YYYYMMDD') === moment(new Date()).format('YYYYMMDD')) {
```

Comment:
This can probably be written as a oneliner: `return (moment(date).format('YYYYMMDD') === moment(new Date()).format('YYYYMMDD'));`